udata_gouvfr/templates/faq/third-party.html

```diff
@@ -0,0 +1,82 @@
+</code></pre>
+
+<p>En remplaçant l’<code>IDENTIFIANT DU JEU DE DONNÉES</code> par l’identifiant disponible sur sa page dédiée vous devriez voir apparaître sur votre site un cartouche contenant les informations relatives à ce jeux de données de la manière suivante :</p>
+
+<p>[image]</p>
+
+<p>Il est possible d’intégrer plusieurs jeux de données à la fois en dupliquant la ligne correspondant à l’élément <code>&lt;div&gt;</code> avec un nouvel identifiant.</p>
+
+<h2>2. Intégration pour une organisation ou un territoire</h2>
+
+<p>Le mécanisme pour afficher tous les jeux de données relatifs à une organisation ou un territoire est le même que celui pour l’intégration d’un seul jeu de donnée décrit précédemment.</p>
+
+<p>Cet usage est recommandé si vous êtes responsable de cette organisation ou de ce territoire et souhaitez faire la promotion des jeux de données associés. Par exemple dans la cadre de la loi Notre, il est demandé aux communes de plus de 3500 habitants de rendre accessible en open data certaines données. L’usage de la plateforme <a href="https://www.data.gouv.fr/fr/">data.gouv.fr</a> pour déposer ces données puis les intégrer sur son propre site est rendue possible par cette solution.</p>
+
+<h3>2.1 Intégration d’une organisation</h3>
+
+<pre><code class="html">&lt;div data-udata-organization="IDENTIFIANT DE L’ORGANISATION"&gt;&lt;/div&gt;
+&lt;script src="https://www.data.gouv.fr/static/widgets.js" id="udata" async defer onload="udataScript.loadOrganization()"&gt;&lt;/script&gt;
+</code></pre>
+
+<p>En remplaçant l’<code>IDENTIFIANT DE L’ORGANISATION</code> par l’identifiant disponible sur sa page dédiée vous devriez voir apparaître sur votre site un cartouche contenant les informations relatives aux jeux de données de cette organisation de la manière suivante :</p>
+
+<p>[image]</p>
+
+<p>Optionnellement, il est possible d’afficher une barre de recherche pour laisser la possibilité au visiteur de filtrer la liste des jeux de données affichés. Cela est activé en passant l’option <code>{withSearch: true}</code> à la méthode <code>loadOrganization()</code> ci-dessus.</p>
+
+<h3>2.1 Intégration d’un territoire</h3>
+
+<pre><code class="html">&lt;div data-udata-territory="IDENTIFIANT DU TERRITOIRE"&gt;&lt;/div&gt;
+&lt;script src="https://www.data.gouv.fr/static/widgets.js" id="udata" async defer onload="udataScript.loadTerritory()"&gt;&lt;/script&gt;
+</code></pre>
+
+<p>En construisant l’<code>IDENTIFIANT DU TERRITOIRE</code> qui doit être de la forme <code>fr-town-13004</code> où <code>13004</code> est le code INSEE de la commune (attention il diffère du code postal). Vous pouvez aussi récupérer les jeux de données relatifs à un département <code>fr-county-13</code> ou une région <code>fr-region-93</code>. Vous devriez alors voir apparaître sur votre site un cartouche contenant les informations relatives aux jeux de données de ce territoire de la manière suivante :</p>
```

Comment:
I put ids within URL as GET parameters and I don't want to deal with potentially badly encoded slashes.